CMakeLists.txt

```diff
@@ -7,6 +7,22 @@ set(OPENSCAP_VERSION_PATCH "0")
 set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}")
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
 
+message(STATUS "OpenSCAP ${OPENSCAP_VERSION}")
+message(STATUS "(see ${CMAKE_SOURCE_DIR}/README.md for build instructions)")
+message(STATUS " ")
+
+# Strictly speaking in-source will work but will be very messy, let's
+# discourage our users from using them
+if ("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
```

Comment:
As a user, I would prefer to see the errors right away, not having to scroll up in order to see why the configuration has failed. Right now, the configuration is very fast, and this error message related to it is neither at the beginning nor at the bottom of the output.
There are also too many newlines in the error message that is print to the screen, but it seems that nothing can be done about that.

Reply:
I straight up stole this from SSG :-) I have made a PR there, check it out please. If it's accepted I can do the same here.

https://github.com/OpenSCAP/scap-security-guide/pull/2540

Reply:
The SSG PR was accepted so I have done the same here as well. It fatal errors now.

Reply:
I have tried that one, it works nicely.